src/commonMain/kotlin/exchange.dydx.abacus/processor/router/skip/SkipProcessor.kt

```diff
@@ -60,6 +63,25 @@ internal class SkipProcessor(
         return modified
     }
 
+    override fun receivedEvmSwapVenues(
+        existing: Map<String, Any>?,
+        payload: Map<String, Any>
+    ) {
+        val venues = parser.asNativeList(payload.get("venues"))
+        val evmSwapVenues = venues?.filter {
+            parser.asString(parser.asMap(it)?.get("name"))?.endsWith(UNISWAP_SUFFIX) == true
+        }?.map {
+            val swapVenue = parser.asMap(it)
+            mapOf(
+                "name" to parser.asString(swapVenue?.get("name")),
+                "chain_id" to parser.asString(swapVenue?.get("chain_id")),
```

Comment:
am aware that in a different time the recommended pattern would be to create a processor. it just feels too heavy weight fro something this simple